app/views/recipes/_recipe.html.erb

```diff
@@ -0,0 +1,83 @@
+<div id="<%= dom_id recipe %>">
+  <p>
+    <h2 class="title is-2"><%= recipe.name %></h2>
+  </p>
+  <div class="box">
+    <article class="media">
+      <div class="media-content">
+        <div class="content">
+          <p>
+            <strong>Preparation time:</strong>
+            <%= recipe.preparation_time %>
+          </p>
+          <p>
+            <strong>Cooking time:</strong>
+            <%= recipe.cooking_time %>
+          </p>
+          <p><strong>Steps go here...</strong></p>
+        </div>
+        <br />
+        <nav class="level">
+          <div class="level-left is-mobile">
+              <%= link_to recipe, class: "level-item" do %>
+                <button class="button is-save is-outlined is-rounded is-responsive">Generate shopping list</button>
+              <% end %>
+          </div>
+          <div class="level-right is-mobile">
+              <!--%= link_to recipe_new_ingredient_path(recipe.id), class: "level-item", method: :get do %-->
+              <%= link_to recipe_new_ingredient_path(recipe.id), class: "level-item", method: :get do %>
+                <button class="button is-save is-outlined is-rounded is-responsive">Add ingredient</button>
+              <% end %>
```

Comment:
- [ ] The functionality to add ingredients to a recipe is not working as expected. Clicking on "Add Ingredient" simply reloads the page but the ingredient is never added to the recipe
![image](https://user-images.githubusercontent.com/46839548/212499556-24532486-871a-4555-a635-2fa042fb0f9d.png)